parsing/intermediate/dialects/dialect1/parserImpls.ftl

```diff
@@ -4079,6 +4079,8 @@ SqlAlter SqlAlter() :
         alterNode = SqlAlterTable(s, scope)
     |
         alterNode = SqlSetOption(s, scope)
+    |
+        alterNode = SqlAlterProcedure(s, scope)
```

Comment:
nit: Can we put these clauses in alphabetical order?

Reply:
Updated